### PR TITLE
Bug 1275468: use the query-string instead of the hash-fragment for search parameters

### PR DESCRIPTION
**Pull request by globau**
does the bare minimum to switch from storing the search parameter in the hash to the query-string.

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -3,17 +3,25 @@ if (!window.console) {
...
+    var qs = window.location.href;
+    var i = qs.indexOf("?");
+    return i == -1 ? "" : qs.substring(i);
+  }
+}
+
+// Poll for changes to the query string
 (function() {
-  var hash = window.location.hash;
+  var qs = window.location.moz_queryString();
```
Comment (tofumatt) [inline js/common.js]: I know the code isn't great to begin with, but can you use full variable names so this is easy to read? `queryString` is easier to scan than `qs`.

File: js/common.js [hunk 2]
```diff
@@ -228,9 +236,15 @@ var SearchManager = {
   },
 
   onLoad: function onLoad() {
+    var search;
```
Comment (tofumatt) [inline js/common.js:239]: Hah, thanks 😄

**Comment by tofumatt** [comment]
r+wc, let me know if you don't have merge access and I'll get this merged once the variable names are tweaked, thanks!

**Comment by globau** [comment]
my github-fu is rusty; if i've done it right this should be ok to merge now (i don't have merge access).

**Comment by tofumatt** [comment]
No worries (also that was perfect!)

These days we can squash and merge but thanks for the single commit still 😄

**Comment by tofumatt** [comment]
Note that this will need a deploy of the phonebook to work. I'll do that, just commenting here to remind myself.

**Comment by tofumatt** [comment]
Actually, just watch https://bugzilla.mozilla.org/show_bug.cgi?id=1288757, that will deploy master and get this change in as well. Our phonebook deployment setup is retro 😭

**Pull request merged**